flamingpy/utils/viz.py

```diff
@@ -33,6 +33,7 @@
 
 import numpy as np
 import networkx as nx
+import plotly.graph_objects as go
```

Comment:
If you add additional packages, make sure to include them in the requirements file

Reply:
https://github.com/XanaduAI/flamingpy/blob/main/dev_requirements.txt